### PR TITLE
DDBEASY-251 - Dont make overlay appear if CTRL is clicked.

### PR DESCRIPTION
**Pull request by guddo**
http://platform.dandigbib.org/issues/1333

### CODE REVIEW

File: js/ting_search_overlay.js [hunk 1]
```diff
@@ -5,23 +5,36 @@
...
-      // Overlay not found so create is and display.
-      overlay = $('<div class="search-overlay--wrapper"><div class="search-overlay--inner"><i class="icon-spinner icon-spin search-overlay--icon"></i><p class="search-overlay--text">' + Drupal.t('Searching please wait...') + '</p><p class="cancel"><a href="#">' + Drupal.t('Cancel') + '</a></p></div></div>');
-      $('body').prepend(overlay);
-    }
-    else {
-      // Overlay found.
-      if (typeof remove_overlay !== 'undefined') {
-        // If toggle remove it.
-        overlay.remove();
+    if (ctrlKeyIsPressed === false) {
```
Comment (cableman) [inline js/ting_search_overlay.js]: Should ctrlKeyIsPressed variable be reset - so another search in the window after a ctrl click gets the overlay?

**Comment by guddo** [comment]
http://codepen.io/alexduloz/pen/nteqG/

File: js/ting_search_overlay.js (hunk 1 above)
Comment (guddo) [inline js/ting_search_overlay.js]: Fixed in new commit, which also supports OSX cmd-button.

**Comment by kasperg** [comment]
Reviewed and approved. This is ready for merging.

Thanks!

**Pull request merged**